client/app/telecom/pack/xdsl/modem/templates/config/pack-xdsl-modem-templates-config.html

```diff
@@ -0,0 +1,246 @@
+                                    data-property="frequency" data-sortable>
+                            {{$row.frequency}}
+                        </oui-column>
+                        <oui-column data-title="'xdsl_modem_template_config_wifi_guest' | translate"
+                                    data-property="guest" data-sortable>
+                            <span data-translate="{{ $row.guest ? 'common_yes_capitalized' : 'common_no_capitalized' }}">
+                            </span>
+                        </oui-column>
+                    </oui-datagrid>
+                </div> <!-- End of Wifi configuration -->
+
+                <!-- DMZ IP and MTU size -->
+                <div class="form-inline">
+                    <h5 data-translate="xdsl_modem_template_config_dmz_and_mtu"></h5>
+                    <div class="form-group">
+                        <oui-checkbox data-model="$ctrl.parametersToIgnore.dmzIP" data-disabled="$ctrl.isDisabled.dmzIP">
+                            <span data-translate="xdsl_modem_template_config_ignore_dmz"></span>
+                        </oui-checkbox>
+                        <oui-field label="{{'xdsl_modem_template_config_ip_dmz' | translate}}">
+                            <input class="oui-input" type="text" id="dmzIp" name="dmzip" ng-model="$ctrl.template.detail.dmzIP" disabled>
+                        </oui-field>
+                    </div>
+                    <div class="form-group">
+                        <oui-checkbox data-model="$ctrl.parametersToIgnore.mtuSize" data-disabled="$ctrl.isDisabled.mtuSize">
+                            <span data-translate="xdsl_modem_template_config_ignore_mtu"></span>
+                        </oui-checkbox>
+                        <oui-field label="{{'xdsl_modem_template_config_mtu' | translate}}">
+                            <input class="oui-input" type="text" id="mtuSize" name="mtuSize" ng-model="$ctrl.template.detail.mtuSize" disabled>
+                        </oui-field>
+                    </div>
+                </div> <!-- End of DMZ IP and MTU size -->
+
+                <!-- Port mapping configuration -->
+                <div class="form-group">
+                    <h5 data-translate="xdsl_modem_template_config_port_mapping"></h5>
+                    <oui-datagrid data-rows="$ctrl.template.detail.portMapping" data-page-size="5">
+                        <oui-column data-title="'xdsl_modem_template_config_ignore' | translate"
+                                    data-property="ignore">
+                            <oui-checkbox data-model="$row.ignore" data-on-change="$ctrl.onRowSelectPortMapping($row.name)"></oui-checkbox>
+                        </oui-column>
+                        <oui-column data-title="'xdsl_modem_template_config_port_mapping_name' | translate"
+                                    data-property="name" data-sortable="asc">
+                            {{$row.name}}
```

Comment:
Please don't use this notation, instead use
```suggestion
                            <span data-ng-bind=":: $row.name"></span>
```